features/fork.feature

```diff
@@ -164,6 +164,22 @@ Scenario: Related fork already exists
      Then the exit status should be 0
      And the url for "mislav" should be "git@github.com:mislav/dotfiles.git"
 
+   Scenario: Related fork and related remote, but with differing protocol, already exist
+       Given the "mislav" remote has url "https://github.com/mislav/dotfiles.git"
```

Comment:
If the remote was created using https protocol, then `hub fork` could fail even if the remote already exists pointing to the correct forked repo. This could be corrected with some more commits converting https:// github urls to ssh equivalents, which I believe be very simple.